index.html

```diff
@@ -9,19 +9,35 @@
   <title> Portfolio tite</title>
   <link rel="stylesheet" href="styles.css">
   <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
+  <script src="script.js" defer></script>
 </head>
 
 <body>
   <header class="navtools_container">
+   
     <h2 class="logo">Tite</h2>
+    <ul class="mobile-menu">
+      <li class="mobile-items">
+        <a href="#home" class="mobile-link">Portfolio</a>
+      </li>
+      <li class="mobile-items">
+        <a href="#about" class="mobile-link">About</a>
+      </li>
+      <li class="mobile-items">
+        <a href="#contact" class="mobile-link">Contact</a>
+      </li>
+    </ul>
+
+      <div class="hamburger navbar">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+

```

Comment:
- [ ] Please stick to the Figma design your design is not the same as the Figma design. If you check below :point_down: the comparison between your design and Figma design you will see clearly your `cross` icon and the Figma `cross` icon spacing are not the same so kindly fix this design issue also :+1: 

**Your Design**
![Screenshot from 2023-04-18 03-28-30](https://user-images.githubusercontent.com/45226848/232624617-0c53269b-2315-4117-b965-fa3c0b8a896c.png)

**Figm Design** 
![Screenshot from 2023-04-18 03-28-19](https://user-images.githubusercontent.com/45226848/232624625-cb09bc85-ddfe-4676-a6e1-faa50b44196a.png)